### DISCUSSION TITLE
migrate dev after squash is asking to reset the db

### DISCUSSION THREAD

**Discussion by halindraprakoso** [prisma/prisma#14458] [Q&A] [answered]
hello,

I squashed the migrations following this guide: 
https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/squashing-migrations#how-to-create-a-clean-history-in-a-production-environment

now that I have only one migration (the squashed_migrations), I updated the schema and executed `prisma migrate dev`. However it asked me to reset the db since the migrations prior to the squash was not found. I expected that it would simply created a new migration on top of the squash. I also already ran `prisma migrate resolve --applied squashed_migrations` on the dev db as well, but it was still the same. 

am I misunderstanding it or did I miss a step?

**Comment by austincrim** [comment, marked as answer]
Hey @halindraprakoso 👋,

Could you share the exact error message?

[This documentation fix](https://github.com/prisma/docs/pull/3447) might be related, but I'm not sure yet. You could try prefixing your squashed_migration with zeros and trying again.

**Reply by halindraprakoso** [reply]
ah I see. thanks Austin. Unfortunately I decided to reset it anyway since it was early in dev and was not a huge problem. However I can vaguely remember and guess that that documentation fix could be what I needed.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
